internal/config/settings.go

```diff
@@ -176,6 +176,7 @@ var defaultCommonSettings = map[string]interface{}{
 	"basename":       false,
 	"colorcolumn":    float64(0),
 	"cursorline":     true,
+	"diffgutter":     true,
```

Comment:
I think this should be `false` by default. I'm open to hearing other opinions though.

Reply:
I'd prefer having it `true` by default as in most other editors out there.

Reply:
I think it should be disabled by default. I feel like micro should be as simple as possible without configuration. Thought that's my philosophy.

Reply:
I believe Micro should be as *useful* as possible without configuration. 99% of programmers use Git, and this feature provides an immediate benefit to them (not having to run `git diff` in order to see changes) without negatively impacting anyone else. I am not aware of any editor that has a built-in diff gutter and disables it by default.

I have carefully designed this feature to be as unobtrusive as possible even for people who don't need/want it. The gutter occupies only a single cell of horizontal space. The impact on CPU and memory usage should be minimal (see above). I definitely think this should be a default feature, especially since (to my knowledge) Micro will be the first terminal-based text editor with a built-in diff gutter and displaying this capability by default could raise its popularity. It definitely was one of the first things I missed when I tried out Micro.